docs/tutorial.rst

```diff
@@ -826,34 +853,25 @@ string instead of a proper Python boolean, we'll get an error:
         raise exceptions.MetadataValidationError from ve
     tskit.exceptions.MetadataValidationError
 
-The metadata on the rows can be accessed directly:
-
-.. code-block:: python
-
-    tables.individuals[0].metadata["accession"]
-
-::
-
-    'Bob1234'
-
-Both the schema and the metadata can be retrieved from a :class:`tskit.TreeSequence`:
+To set the top-level metadata, just assign it. Validation and encoding happen as specified by the
+top-level metadata schema (Note that provenance info should go in :ref:`sec_provenance`
+not metadata):
 
 .. code-block:: python
 
-    ts = tables.tree_sequence()
-    ts.table_metadata_schemas.individual.schema
-
-::
+    ts.metadata = {"mean_coverage": 200.5}
 
-    {'codec': 'json', 'type': 'object', 'properties': {'accession': {...}}}
+To modify a schema, for example to add a key; first get the dict representation, modify
+then write back:
 
 .. code-block:: python
 
-    ts.individual(0).metadata["accession"]
+    schema_dict = tables.individuals.metadata_schema.schema
+    schema_dict["properties"]["newKey"] = {"type": "string"}
+    tables.individuals.metadata_schema = tskit.MetadataSchema(schema_dict)
 
-::
-
-    'Bob1234'
+To modify the metadata of rows in tables use the :ref:`sec_tutorial_metadata_bulk`.
+to
```

Comment:
missing end of the sentence?

Reply:
Fixed in https://github.com/tskit-dev/tskit/pull/741/commits/004263ac791fbc3b4dd812ba716272b473a30f73